docs/termux.md

```diff
@@ -10,40 +10,44 @@
   - [Create a new user](#create-a-new-user)
   - [Install Go](#install-go)
   - [Install Python](#install-python)
+  - [Working with PRoot](#working-with-proot)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 ## Install
 
 1. Get [Termux](https://f-droid.org/en/packages/com.termux/) from **F-Droid**.
-2. Install Debian by running the following.
+2. Install Debian by running the following:
    - Run `termux-setup-storage` to allow storage access, or else code-server won't be able to read from `/sdcard`.\
-      If you used the Andronix command then you may have to edit the `start-debian.sh` script to mount `/sdcard` just as simple as uncommenting the `command+=" -b /sdcard"` line.
-     > The following command was extracted from [Andronix](https://andronix.app/) you can also use [proot-distro](https://github.com/termux/proot-distro).
+     > The following command is from [proot-distro](https://github.com/termux/proot-distro) you can also use [Andronix](https://andronix.app/).
      > After Debian is installed the `~ $` will change to `root@localhost`.
 
 ```bash
-pkg update -y && pkg install wget curl proot tar -y && wget https://raw.githubusercontent.com/AndronixApp/AndronixOrigin/master/Installer/Debian/debian.sh -O debian.sh && chmod +x debian.sh && bash debian.sh
+pkg up -y && pkg in proot-distro -y && proot-distro install debian && proot-distro login debian
```

Comment:
maybe a silly question but is the `in ` supposed to be there? (I don't use `pkg` much)

Reply:
Oh fair.  pkg in and up are shorthand for install and update.

Reply:
Updated in a subsequent commit.  Thanks!